### ISSUE TITLE
Angular 11.1 LESS not building issue

### ISSUE THREAD

**Issue by dylanvdmerwe** [NG-ZORRO/ng-zorro-antd#6368]
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-ivy-pu6fe9?file=package.json](https://stackblitz.com/edit/ng-zorro-antd-ivy-pu6fe9?file=package.json)

### Steps to reproduce
Upgrade to Angular 11.1.0.

### What is expected?
The project should compile correctly.

### What is actually happening?
There is a ModuleBuildError caused by a problem with size.less in the table syles.

```
Error: ./src/theme.less
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/less-loader/dist/cjs.js):


// ================================================================
.table-size(~'middle', @table-padding-vertical-md, @table-padding-horizontal-md, @table-font-size-md);
^
Cannot read property 'numerator' of undefined
      Error in C:\Projects\WunderlustAPI\WebsiteProjects\Website.Site\wunderlust-app\node_modules\ng-zorro-antd\table\style\size.less (line 53, column 0)
    at C:\Projects\WunderlustAPI\WebsiteProjects\Website.Site\wunderlust-app\node_modules\webpack\lib\NormalModule.js:316:20
    at C:\Projects\WunderlustAPI\WebsiteProjects\Website.Site\wunderlust-app\node_modules\loader-runner\lib\LoaderRunner.js:367:11
    at C:\Projects\WunderlustAPI\WebsiteProjects\Website.Site\wunderlust-app\node_modules\loader-runner\lib\LoaderRunner.js:233:18
    at context.callback (C:\Projects\WunderlustAPI\WebsiteProjects\Website.Site\wunderlust-app\node_modules\loader-runner\lib\LoaderRunner.js:111:13)
    at Object.lessLoader (C:\Projects\WunderlustAPI\WebsiteProjects\Website.Site\wunderlust-app\node_modules\less-loader\dist\index.js:57:5)
```

| Environment | Info |
|---|---|
| ng-zorro-antd | 11.0.2 |
| Browser | Chrome |

**Comment by vthinkxie** [comment]
duplicated of #6359, you can downgrade less to 3.x as a temp solution.

should be fixed via #6365

**Comment by karenengineer** [comment]
@vthinkxie Hey, i have this issue, downgrading less is doesn't help me.

./src/theme.less - Error: Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/@angular-devkit/build-angular/node_modules/less-loader/dist/cjs.js):
